src/runtime/cuda/cuda_hardware_manager.cpp

```diff
@@ -219,6 +219,15 @@ cuda_hardware_context::get_property(device_uint_property prop) const {
   case device_uint_property::max_global_size2:
     return _properties->maxThreadsPerBlock * _properties->maxGridSize[2];
```

Comment:
It seems that `device_uint_property::max_global_sizeX` are not correct (should be `_properties->maxThreadsDim[X] * _properties->maxGridSize[X]). However, as far as I can tell, they are not used now, so we can safely remove them.

Reply:
Done.